Khaled pid mods

leavesandlemmas opened the pull request:
Line 102, 103: SHould those variables be renamed since they aren't the maximum / minimum according to the comment?

leavesandlemmas committed:
Update VPD-norain-plot.cr1

leavesandlemmas committed:
Update VPD-norain-plot.cr1

leavesandlemmas committed:
Update VPD-norain-plot.cr1

justinmcgrath committed:
Format comments

justinmcgrath committed:
Format comments.

justinmcgrath commented:
It would be nice to have other names, but I can't think of anything. If we describe the equation in the code, we could use something generic, like Greek letters. I'm not worried about it though.

Using averages for a couple of the new variables doesn't seem like it will work well. It seems more useful to sample ixKi_sum and P_error. Not sure about it tough. It's not critical, so let's sample those and we can change later if needed.

justinmcgrath committed:
Sample errors.

justinmcgrath committed:
Add voltage limiter placeholder.

leavesandlemmas committed:
Update VPD-norain-plot.cr1

+ Added undeclared variable to variable declaration preamble
+ Removed delta_t from voltageLimiter function; variable not used in function body.

leavesandlemmas committed:
Update VPD-norain-plot.cr1

leavesandlemmas committed:
Update VPD-norain-plot.cr1

justinmcgrath commented:
I think this is what is running on the loggers now. If that's correct, please merge this into main and we'll base further changes on that. If this is not what's on the loggers, what do they have that is different?

leavesandlemmas commented:
Only thing I have on this PR which isn't on the datalogger is removing the "set integral to zero if temp diff. is too high". I
ll move it too a new branch

justinmcgrath commented:
That's not committed here. So then this should be what's on the loggers.
